tests/conftest.py

```diff
@@ -48,17 +50,38 @@ def seed_application_records(
             app["fund_id"] = random_fund_id
             app["round_id"] = random_round_id
         app_flags = []
+        app_flags_v2 = []
         if "flags" in app:
             app_flags = app["flags"]
             app.pop("flags")
+        app_flags_v2 = []
+        if "flags_v2" in app:
+            app_flags_v2 = app["flags_v2"]
+            app.pop("flags_v2")
```

Comment:
`pop` looks to return the value so could maybe do this in one line
```suggestion
            app_flags_v2 = app.pop("flags_v2")
```

Reply:
Done 👍